File: OOP_Ponavljanje/php/staticPrimjer.php
<?php 

//definicija klase
class Student {
//podatkovni clanovi
	protected $ime;
	private $matBr;

//staticki clan klase
	public static $num;

//konstruktor
	public function __construct($ime, $matBr) {
		Student::$num++;
		$this->ime = $ime;
		$this->matBr = $matBr;
		echo "Trenutni broj studenata: " . Student::$num . "<br/>";
	}

	public static function dohvatiNesto(Student $obj) {
		$obj->getMatBr();
	}

//javna metoda
	public function getMatBr() {
		return $this->matBr;
	}

//destruktor klase
	public function __destruct() {
		echo $this->ime . " I am destroyed!" . "<br />";
	}
}

$pero = new Student("Petar", 1234);
$ivana = new Student("Ivana", 123123);

// $mate = new Student("Mate", 112233);

// $mate = $pero;
// echo $mate->getMatBr() . "<br />";

File: OOP_Ponavljanje/c++/staticPrimjer.cpp
#include<iostream>
using namespace std;

class Student {
    int matBr;
    static int numObj;
public:
    
    string ime;
    Student(string ime, int matBr) {
        Student::numObj++;
        this->ime = ime;
        this->matBr = matBr;
    }
    
    static int getNumObj() {
        return Student::numObj;
    }
    
};
int Student::numObj = 0;

int main(int argc, char** argv) {
    Student* pero = new Student("Petar", 1234);
    Student* mate = new Student("Mate", 3432);
    cout << Student::getNumObj();
    return 0;
}